templates/master/00-master/_base/files/usr-local-bin-etcd-member-remove-sh.yaml

```diff
@@ -0,0 +1,42 @@
+filesystem: "root"
+mode: 0755
+path: "/usr/local/bin/etcd-member-remove.sh"
+contents:
+  inline: |
+    #!/usr/bin/env bash
+
+    # example
+    # sudo ./etcd-member-recover.sh $etcd_name
```

Comment:
minor: example still says _recover_ instead of _remove_.

Reply:
fixed